automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -146,10 +148,21 @@ def satellite6_capsule_upgrade(admin_password=None):
         print('Wrong Capsule Version Provided to upgrade to. '
               'Provide one of 6.1, 6.2')
         sys.exit(1)
+    if not os.environ.get('CAPSULE_SUBSCRIPTION') and not os.environ.get(
+            'CAPSULE_AK'):
+        print('One of Env. Var CAPSULE_SUBSCRIPTION or CAPSULE_AK should be'
+              'defined!')
+        sys.exit(1)
     if admin_password is None:
         admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
     # Setting Capsule61 Repos
     major_ver = distro_info()[1]
+    # Re-register Capsule for 6.2¬
```

Comment:
AS per host unification feature: if there is a host registered where the Host and Content Host are in different organizations (e.g. host not in org, and content host in one), the content host will be unregistered as part of the upgrade process. 

And in 6.1, generally when we register capsule, org get associated with capsule content-host but not with host under "All Host". So this re-registration is required.
A thread is already started internally on list on same.